docs/plugins/repository-s3.asciidoc

```diff
@@ -62,16 +66,17 @@ bin/elasticsearch-keystore add s3.client.default.access_key
 bin/elasticsearch-keystore add s3.client.default.secret_key
 ----
 
-The following are the available client settings. Those that must be stored in the keystore
-are marked as `Secure`.
+The following list contains the available client settings. Those that must be
+stored in the keystore are marked as "secure"; all other settings can be
```

Comment:
"can" implies (to me) that this is optional. Yet the secure and non secure settings are disjoint; they cannot be present in the other location. I'm not sure what word would better relate that requirement, though...

Reply:
Thanks for the feedback!  I changed it to "all other settings belong in ..."